crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -90,7 +87,7 @@ impl Protobuf for merkle::Proof {
 pub struct RollupId {
```

Comment:
`RollupId` is actually a concept that we have a wiretype for. For consistency, we should fix this serde derivation and point it to `astria.primitive.v1.RollupId`.

Reply:
After offline discussion, agreed to leave this as is due to our usage of `RollupId`s as keys in maps which are JSON-encoded using the serde trait.  This fails if using the wiretype serialization since the JSON map requires the key to be a string, and making the suggested change means a JSON-encoded `RollupId` is a JSON object rather than a string.